docs/Tutorials/Orientation.md

```diff
@@ -2,50 +2,60 @@
-with one another for periodic boundary conditions. That is, that no block has
-itself as a neighbor. If you wish to additionally identify faces of the same
-block with each other, that must be done in an additional step. This step is
-explained in the "Setting Periodic Boundary Conditions" section.
-
-### Local Corner Numbering Scheme (Local CNS)
-The orientation of the logical axes in each Block define an ordered set of
-corners. Once the logical axes in each Block are determined, it is possible
-to write down a Local CNS:
-
-Each Block has \f$2^{dim}\f$ local corners, numbered from \f$0\f$ to
-\f$2^{dim} - 1\f$. We will take the three-dimensional cube as an example:
-This Block has 8 local corners (regardless how many of these are
-shared), numbered from 0 to 7. The corners are labelled according to the
-CoordinateMap corresponding to the Block, as follows:
+# OrientationMap {#tutorial_orientations}
+
+### Introduction
+Each element in a domain has a set of internal directions which it uses
+for computations in its own local coordinate system. These are referred to
+as the logical directions \f$\xi\f$, \f$\eta\f$, and \f$\zeta\f$, where
+\f$\xi\f$ is the first dimension, \f$\eta\f$ is the second dimension, and
+\f$\zeta\f$ is the third dimension. In a
+domain with multiple elements, the logical directions are not necessarily
+aligned on the interfaces between two elements, as shown in the figure below.
+As fluxes need to be communicated across the boundaries of adjacent elements,
+there needs to be a system in place that organizes the relative orientations of
+elements with respect to their neighbors. This is handled by OrientationMap.
+
+### %OrientationMaps between %Blocks
+Each Block in a Domain has a set of BlockNeighbors, which each hold an
+OrientationMap. In this scenario, the Block is referred to as the host, and
+the OrientationMap held by each BlockNeighbor is referred to as "the
+orientation the host Block has with respect to this BlockNeighbor." This is
+a convention, so we give an example of constructing and assigning the correct
+OrientationMaps:
+
+\image html OrientationMap.png "Two neighboring blocks."
+
+In the image above, we see a domain decomposition into two Blocks, which have
+their logical axes rotated relative to one another. With the left block as
+the host Block, we see that it has a BlockNeighbor in the \f$+\xi\f$ direction.
+The OrientationMap held by the BlockNeighbor determines the mapping from each
+logical direction in the host Block to that in the neighoring Block. That is,
+the OrientationMap takes as input local information and returns neighbor
+information. In this case, the \f$+\xi\f$, \f$+\eta\f$, \f$+\zeta\f$ directions
+in the host map to the \f$+\zeta\f$, \f$+\xi\f$, \f$+\eta\f$ directions in the
+neighbor, respectively. This BlockNeighbor thus holds the OrientationMap
+constructed with the list (\f$+\zeta\f$, \f$+\xi\f$, \f$+\eta\f$). With the
```

Comment:
This doesn't help because I don't know how an OrientationMap constructor works.  How about adding a sentence (add the sentence after the "the OrientationMap takes as input...") that says how OrientationMaps are constructed.  Something like "An OrientationMap is constructed by passing in the BlockNeighbor directions that correspond to the \f$+\xi\f$, \f$+\eta\f$, \f$+\zeta\f$ host directions".